.github/workflows/pr-build.yml

```diff
@@ -58,10 +58,18 @@ jobs:
           repository: ${{github.event.pull_request.head.repo.full_name}}
           fetch-depth: 0
 
-      - name: verify-protos
-        run: |
-          buf lint
-          buf breaking --against-input "https://github.com/${GITHUB_REPOSITORY}.git#branch=main"
+      - name: Setup buf
+        uses: bufbuild/buf-setup-action@v1.9.0
+        with:
+          github_token: ${{ github.token }}
+
+      - name: Lint protos
+        uses: bufbuild/buf-lint-action@v1
+
+      - name: Check for breaking changes
+        uses: bufbuild/buf-breaking-action@v1
```

Comment:
These pre-packaged actions will provide better messaging/annotations directly on the PR.